src/settings_gui.cpp

```diff
@@ -365,6 +436,24 @@ struct GameOptionsWindow : Window {
 		this->reload = false;
 		this->gui_scale = _gui_scale;
 
+		// Fonts sizes could be zero here. The font loading code will choose a non-zero value but then it scales it making it useless here.
```

Comment:
Wrong comment format, and it would be better to explain why this code exists instead of explaining what it doesn't do. The negation is confusing. 🙂